src/mvba/mod.rs

```diff
@@ -13,9 +17,19 @@ mod vcbc;
 
 pub type NodeId = usize;
 
-/// A proposed data with the proof inside. It is the same as $(w, π)$ in the spec.
+/// A proposed data. It is the same as $w$ in the spec.
 pub type Proposal = Vec<u8>;
 
+/// A prof if decided proposed data. It is the same as $π$ in the spec.
+#[derive(Debug)]
+pub struct Proof {
+    pub domain: Domain,
+    pub proposer: NodeId,
+    pub abba_signature: Signature,
+    pub abba_round: usize,
+    pub vcbc_signature: Signature,
+}
```

Comment:
Is it possible to write a a verify function on this proof that just takes the public key set as argument?

```rust
impl Proof {
    fn verify(&self, pks: &PublicKeySet) -> bool {
        ...
    }
}
```

This proof will be sent to nodes who don't have a consensus instance available (i.e. think clients or non-elder nodes). We still want them to be able to validate that the proof was created through consensus but without having to create a consensus instance.